cmd/lncli/commands.go

```diff
@@ -2168,9 +2168,17 @@ func sendToRoute(ctx *cli.Context) error {
 			"from incoming array of routes: %v", err)
 	}
 
+	if len(routes.Routes) == 0 {
+		return fmt.Errorf("no routes provided")
+	}
+
+	if len(routes.Routes) > 1 {
```

Comment:
shouldn't this rather error?

Reply:
Changes to cli reverted as the functionality has now been implemented in the sub server.

Reply:
converted to error